ducktape/cluster/remoteaccount.py

```diff
@@ -315,20 +315,22 @@ def ssh_capture(self, cmd, allow_fail=False, callback=None, combine_stderr=True,
 
         def output_generator():
 
+            stdout_lines = []
             for line in iter(stdout.readline, ''):
-
+                stdout_lines.append(line)
                 if callback is None:
                     yield line
                 else:
                     yield callback(line)
             try:
                 exit_status = stdout.channel.recv_exit_status()
                 if exit_status != 0:
+                    full_output = "stderr: {}\nstdout: {}".format(stderr.read(), ''.join(stdout_lines))
```

Comment:
My only nit would be if we want to have some sort of (both easily parseable and visually compelling) delineator between these two output streams. E.g.
```
===== stderr ===
...
==== stdout ===
...
```